### ISSUE TITLE
Version for TYPO3 9 and 10

### ISSUE THREAD

**Issue by bihor** [joppnet/jn_lighterbox#9]
It would be nice, if you would make a version for TYPO3 9 (and 10) that contain only new TypoScript conditions:
https://wiki.sebkln.de/doku.php?id=typo3:about:conditions
https://docs.typo3.org/m/typo3/reference-typoscript/master/en-us/Conditions/Index.html
https://www.nitsan.in/de/blog/all-you-need-to-know-about-typoscript-conditions-with-symfony-expression-language/
https://www.clickstorm.de/blog/typoscript-conditions-symfony-expression-language/
https://blog.blue-side.de/2018/10/globalvar-gpl-1-mit-typo3-9-und-sitehandling/
https://www.typo3lexikon.de/typo3-tutorials/typoscript/referenz/condition-syntax.html
https://www.coding.ms/de/how-to/migration-von-typoscript-conditions-zu-typo3-95/

**Comment by oliverschloebe** [comment]
Hi @bihor, it's actually planned already to conditionally include different TypoScript setups based on the TYPO3 version used. ✌️ 

Unfortunately no ETA yet.

**Comment by oliverschloebe** [comment]
> Unfortunately no ETA yet.

Let me rephrase that: ETA is now as I just released version 1.2.0. :-)
